CMakeLists.txt

```diff
@@ -3,6 +3,7 @@
 
 # Minimum CMake required
 cmake_minimum_required(VERSION 3.25)
+cmake_policy(SET CMP0104 OLD)
```

Comment:
should we set the policy to be new?

Reply:
"SET CMP0104 NEW" will initialize CMAKE_CUDA_ARCHITECTURES inside cmake. Since now both cmdclass.py and build.sh will set it explicitly, I am fine to set the policy to be new. 

Just remind that build.bat does not set CMAKE_CUDA_ARCHITECTURES, so build ORT-extensions in Windows with cuda enabled will have some strange value for this variable. 

Tested just now, running "nvidia-smi --query-gpu=compute_cap --format=csv,noheader,nounits" in my win machine has the result 7.5, but "SET CMP0104 NEW" will make it 52. Are you OK with that?